src/Utility/Text.php

```diff
@@ -1037,7 +1037,7 @@ public static function parseFileSize(string $size, mixed $default = false): mixe
             return (int)$size;
         }
 
-        if ($default !== false) {
+        if ($default !== null) {
             return $default;
```

Comment:
We could cast the default to int and keep the return value of the method as `int` only.

Reply:
I just thought someone is using it to find out if the parsing worked. And 0 wouldnt be clear i guess

Reply:
In that case we can make the return type nullable int?

Reply:
hmm.. the docblock has e..g for using string for invalid size, `Value to be returned when invalid size was used, for example 'Unknown type'`

Reply:
I would rather just throw an exception for un-parseable inputs